src/components/Container/Container.tsx

```diff
@@ -0,0 +1,97 @@
+import styled from "styled-components";
+import { HTMLAttributes } from "react";
+import { Orientation } from "@/components";
+
+type AlignItemsOptions = "start" | "center" | "end";
+type GapOptions = "none" | "xxs" | "xs" | "sm" | "md" | "lg" | "xl" | "xxl";
+type GrowthOptions = "0" | "1" | "2" | "3" | "4" | "5" | "6";
+type JustifyContentOptions =
+  | "flex-start"
+  | "flex-end"
+  | "center"
+  | "space-between"
+  | "space-around"
+  | "space-evenly"
+  | "start"
+  | "end"
+  | "left"
+  | "right";
+type PaddingOptions = "none" | "xxs" | "xs" | "sm" | "md" | "lg" | "xl" | "xxl";
+
+export interface ContainerProps extends HTMLAttributes<HTMLDivElement> {
+  alignItems?: AlignItemsOptions;
+  children?: React.ReactNode;
+  fillWidth?: boolean;
+  gap?: GapOptions;
+  grow?: GrowthOptions;
+  hasBorder?: boolean;
+  isResponsive?: boolean;
+  justifyContent?: JustifyContentOptions;
+  orientation?: Orientation;
+  padding?: PaddingOptions;
+}
+
+const Container = ({
+  alignItems = "center",
+  children,
+  fillWidth = false,
+  gap = "none",
+  grow = "1",
+  hasBorder,
+  isResponsive,
+  justifyContent = "start",
+  orientation = "horizontal",
+  padding = "md",
+}: ContainerProps) => {
+  return (
+    <Wrapper
+      $alignItems={alignItems}
+      $fillWidth={fillWidth}
+      $gapSize={gap}
+      $grow={grow}
+      $hasBorder={hasBorder}
+      $isResponsive={isResponsive}
+      $justifyContent={justifyContent}
+      $orientation={orientation}
+      $paddingSize={padding}
+    >
+      {children}
+    </Wrapper>
+  );
+};
+
+const Wrapper = styled.div<{
+  $alignItems: AlignItemsOptions;
+  $fillWidth: boolean;
+  $gapSize: GapOptions;
+  $grow: GrowthOptions;
+  $hasBorder?: boolean;
+  $isResponsive?: boolean;
+  $justifyContent: JustifyContentOptions;
+  $orientation?: Orientation;
+  $paddingSize: PaddingOptions;
+}>`
+  display: flex;
```

Comment:
Should we also add wrapping here?

Reply:
Yep, added, by default `nowrap` with the ability to add `wrap` and `wrap-reverse`.